rest/.jvm/src/main/scala/io/udash/rest/RestServlet.scala

```diff
@@ -47,27 +47,32 @@ class RestServlet(
 
   override def service(request: HttpServletRequest, response: HttpServletResponse): Unit = {
     val asyncContext = request.startAsync()
+    val completed = new AtomicBoolean(false)
+
+    // need to protect asyncContext from being completed twice because
+    // servlet may reuse the same context instance between subsequent requests (not cool)
```

Comment:
nit: `reuse` -> `recycle`? that's the most common terminology regarding this behaviour